saleor/core/utils/__init__.py

```diff
@@ -110,10 +110,10 @@ def get_user_shipping_country(request):
     return request.country
 
 
-def serialize_decimal(obj):
-    if isinstance(obj, decimal.Decimal):
-        return str(obj)
-    return JSONEncoder().default(obj)
+def serialize_amount(obj):
+    if isinstance(obj, Money):
+        return str(obj.amount)
```

Comment:
This looks wrong. If we only want the number to appear in JSON, we should only pass the amount to the serialized structure. This seems to be fixing data at the wrong level.